examples/evm/call-contract-with-token/README.md

```diff
@@ -21,6 +21,11 @@ The aforementioned command pertains to specifying the intended environment for a
 
 An example of its usage is demonstrated as follows: `npm run deploy evm/call-contract-with-token local` or `npm run deploy evm/call-contract-with-token testnet`.
 
+### Test
+`cd` into `evm` folder
+Run the command ` hh test call-contract-with-token/tests/call-contract-with-token.test.js`     
```

Comment:
```suggestion
Run the command `hh test call-contract-with-token/tests/call-contract-with-token.test.js`     
```

Reply:
good catch, i've always used the shorthand but can go to npx hardhat just to make sure it's more accessible (will apply this change to other spots where you left this comment as well)